tools/@aws-cdk/prlint/lint.ts

```diff
@@ -270,15 +270,21 @@ export class PullRequestLinter {
       body,
     });
 
-    // Commenting this code to first test that linter rule works
-    // since this can lead to other PRs closing if not setup correctly
-    // // Closing the PR if it is opened from main branch of author's fork
-    // if (failureMessages.includes(PR_FROM_MAIN_ERROR)) {
-    //   await this.client.pulls.update({
-    //     ...this.prParams,
-    //     state: 'closed',
-    //   });
-    // }
+    // Closing the PR if it is opened from main branch of author's fork
+    if (failureMessages.includes(PR_FROM_MAIN_ERROR)) {
+
+      const errorMessageBody = 'We are unable to accept contributions from `main` branch of a fork. This pull request would be automatically closed now. Please open a new pull request from another branch on your fork. For more information, see https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#step-4-pull-request.'
```

Comment:
This is in case there are multiple linter errors. This would clarify to the author why we are closing the PR.

Reply:
Rico wants something more in line with https://github.com/aws/aws-cdk/pull/27625/files which is why he made that update. Can we just repeat the words there in the closing message?